test/models/pet_test.rb

```diff
@@ -83,6 +83,19 @@ class PetTest < ActiveSupport::TestCase
         assert_not Pet.with_photo.include?(pet_without_image)
       end
     end
+
```

Comment:
If we decide not to show the species for pets without adoptable animals, we can also test that here.